fr/_posts/2024-06-10-boite-outils-durgence.md

```diff
@@ -0,0 +1,100 @@
+---
+altLangPage: /2024/06/10/crisis-toolkit.html
+css:
+-  ../../../css/blog.css
+date: 2024-06-10
+dateModified: 2024-06-10
+description: "Répondre aux besoins des utilisateurs et utilisatrices de Canada.ca en temps de crise."
+title: "Boîte à outils de conception pour les situations d’urgence"
```

Comment:
On the preview the H1 is displaying as: "Boîte à outils de conception pour les situations d’urgence: Blogue de Canada.ca"

Can you please remove ": Blogue de Canada.ca"

Reply:
It was a problem with the CSS, fixed now

Reply:
I still see it it in the H1: https://deploy-preview-163--blogue-sct.netlify.app/2024/06/10/boite-outils-durgence 
![Blog_H1](https://github.com/canada-ca/blogue-canada-ca-blog/assets/118768603/1c8094de-0042-4c73-b972-2d3ef1d89e0c)